tests/contract_testcases/solana/large_exponent.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
+     int256 constant a = 1e1000;
+     int256 constant b = 1e-65537;
+     int256 constant c = 1e65537;
+     function f() private {
+	assembly {
+		let x := 1e65537
```

Comment:
Yul does not allow literals in this notation. There should be an error about it, but you inform only that the exponent is too large.

Reply:
https://docs.soliditylang.org/en/v0.8.12/yul.html#literals

Reply:
ok.

Is there any harm in supporting it in yul?

Reply:
I don't think so.